src/lava/magma/runtime/runtime_services/interfaces.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+import typing as ty
+from abc import ABC, abstractmethod
+
+from lava.magma.compiler.channels.pypychannel import CspRecvPort, CspSendPort
+from lava.magma.core.sync.protocol import AbstractSyncProtocol
+
+
+class AbstractRuntimeService(ABC):
```

Comment:
Missing docstring. What is a RuntimeService? What is it's relation to the Runtime and ProcModels?

Reply:
Will be resolved in another PR via #184, as this would be useful for upcoming release that this change is not a part of.